client/src/services/phases.js

```diff
@@ -26,8 +26,9 @@ const mapPhasesResponse = async (data, columns) => {
   return rowsData;
 };
 
-export const fetchPhases = async () => {
-  const response = await fetch(`${API_URL}/api/v1/phase/`, {
+export const fetchPhases = async (queryString = null) => {
+  const url = queryString ? `${API_URL}/api/v1/phase${queryString}` : `${API_URL}/api/v1/phase/`;
```

Comment:
Necessary? Can't we just remove the trailing forward slash and add ${queryString} at the end? Or would that go `api/v1/phasenull`?

Reply:
`api/v1/phasenull` is exactly what it does! I was also hoping to simplify here

Reply:
alternatively, I could default it to an empty string `queryString = ' ' ` and remove the const url

Reply:
Nah, this is fine!